app/forms/journeys/further_education_payments/provider/omniauth_callback_form.rb

```diff
@@ -19,7 +19,8 @@ def save!
             dfe_sign_in_role_codes: dfe_sign_in_role_codes,
             dfe_sign_in_first_name: dfe_sign_in_first_name,
             dfe_sign_in_last_name: dfe_sign_in_last_name,
-            dfe_sign_in_email: dfe_sign_in_email
+            dfe_sign_in_email: dfe_sign_in_email,
+            claim_started_verified: journey_session.answers.claim.eligibility.verified?
```

Comment:
we cache state of whether claim is verified when journey starts. otherwise chicken and egg occurs after claim becomes verified we would send user to already verified page, this caching prevents that